schema.sql

```diff
@@ -6,7 +6,8 @@ CREATE TABLE articles (
     likes       INT DEFAULT 0,
     views       INT DEFAULT 0,
     color       VARCHAR(20) DEFAULT "#5e9de6",
-    createdAt   TIMESTAMP DEFAULT NOW(), 
+    published   SMALLINT DEFAULT 0,
```

Comment:
Is createdAt not the same as published? Please clarify.

Reply:
Sorry, published is a boolean value represented by 0 or 1. We want this functionality to be able to create an article, but not publish it (won't show for public users).

We want this functionality to be able to save articles that are still a work in progress.

So an article can be created but not published.